multiqc/utils/search_patterns.yaml

```diff
@@ -60,6 +60,8 @@ htseq:
     contents: '__too_low_aQual'
 hicup:
     fn: 'HiCUP_summary_report*'
+jellyfish:
+    fn: '*.hist'
```

Comment:
Ah damn, these ones are the worst - no identifying file contents and a totally user-defined filename. If the `.hist` extension is totally arbitrary and we need to tell people to use a specific filename pattern anyway (or customise their search pattern config), could it be `*_jf.hist` instead? This will need a paragraph in the docs markdown file explaining how it works.

Looking for `.jf` files could be nice in addition? May be a bit of a shot in the dark though.. Not sure if it's worth it.